### DISCUSSION TITLE
Compatibility with kubernetes v1.26

### DISCUSSION THREAD

**Discussion by evertramos** [prometheus-operator/kube-prometheus#2112] [General] [answered]
Hello there!

Is there a target release to be compatible with kubernetes v1.26?

> Thanks for this this greate job!

**Comment by timmow** [comment]
We've always stuck to running from the tagged release branches and are also wondering if there will be a planned release for `1.26` or if its time to switch to running from `main`

**Comment by ash0ash** [comment]
Thanks for raising @evertramos  and @timmow. Just to add our voice to this discussion.
We are also wondering if it is possible to have a tagged release rather than switch to using the `main`  release. This has also been our preferred route for upgrades.
The compatibility matrix has also previously allowed for confirmed compatibility with one previous k8s version. 
We would be really grateful if we could have a tagged release with backwards compatibility of one k8s version for these reasons. Thanks

**Comment by oldNoakes** [comment]
Had the same question and just saw this https://github.com/prometheus-operator/kube-prometheus/pull/2210 - crossing my fingers

**Comment by evertramos** [comment, marked as answer]
For whoever is searching for the specific release for v1.26, here it is:

https://github.com/prometheus-operator/kube-prometheus/tree/release-0.13
